test/utils/image/manifest.go

```diff
@@ -78,7 +78,6 @@ func initReg() RegistryList {
 		GcEtcdRegistry:          "k8s.gcr.io",
 		GcRegistry:              "k8s.gcr.io",
 		SigStorageRegistry:      "k8s.gcr.io/sig-storage",
-		GcrReleaseRegistry:      "gcr.io/gke-release",
```

Comment:
My main question is whether there are any remaining manifests in this repo that reference this? AFK for now, I can take time to try and answer this myself next week

Reply:
Definitely no references other than a CSI test case for GCE (with references in a static yaml file)

Reply:
Should that have been updated?

Reply:
@spiffxp i opened follow ups https://github.com/kubernetes/kubernetes/issues/100274

Reply:
doesn't quite seem to have been the case. https://github.com/kubernetes/kubernetes/pull/100303
left some questions in https://github.com/kubernetes/kubernetes/issues/100274, I don't want to see us depend on this registry but I'd like to understand what we're switching to more.